README.md

```diff
@@ -1,5 +1,7 @@
 # MLSpace
 
+[![Full Documentation](https://img.shields.io/badge/Full%20Documentation-blue?style=for-the-badge&logo=Vite&logoColor=white)](https://awslabs.github.io/mlspace/)
```

Comment:
Where does this documentation get built from?

Reply:
This [github action on merge](https://github.com/awslabs/mlspace/blob/main/.github/workflows/deploy.yml). Previous runs [here](https://github.com/awslabs/mlspace/actions/workflows/deploy.yml).